src/shared/sam/debugger/typescriptSamDebug.ts

```diff
@@ -25,7 +27,38 @@ export async function invokeTypescriptLambda(
     config.samLocalInvokeCommand = new DefaultSamLocalInvokeCommand([WAIT_FOR_DEBUGGER_MESSAGES.NODEJS])
     // eslint-disable-next-line @typescript-eslint/unbound-method
     config.onWillAttachDebugger = waitForPort
-    const c = (await invokeLambdaFunction(ctx, config, async () => {})) as NodejsDebugConfiguration
+    const c = (await invokeLambdaFunction(ctx, config, async () => {
+        // Compile step for TS projects from source code
```

Comment:
might as well put this in a private (e.g. module-scoped) function named something like `onAfterBuild`.